test/conftest.py

```diff
@@ -863,6 +863,7 @@ def get_document_store(
             index=index,
             similarity=similarity,
             use_bm25=True,
+            bm25_parameters={"k1": 1.2, "b": 0.75},  # parameters similar to those of Elasticsearch
```

Comment:
Not a problem but I'm not sure we even need these. Is there any test where you found a large difference?

Reply:
Yes, I found some tests that compare the obtained scores with the expected ones and this helps a little.